workspace/resource_notebook.go

```diff
@@ -171,7 +170,7 @@ const (
 )
 
 func (a NotebooksAPI) recursiveAddPathsParallel(directory directoryInfo, dirChannel chan directoryInfo,
-	answer *syncAnswer, wg *sync.WaitGroup, shouldIncludeDir func(ObjectStatus) bool) {
+	answer *syncAnswer, wg *sync.WaitGroup, shouldIncludeDir func(ObjectStatus) bool, visitor func([]ObjectStatus)) {
```

Comment:
I think this API and `ListParallel` are only used in the exporter. Given that, maybe it makes sense to move them the `exporter` package? That way you can more or less freely make changes to them as needed for exporter, and I don't need to worry about them breaking components in the main provider.

Reply:
yep, I thought about it. But need to look how to make it correctly

Reply:
I didn it already